underlay/src/test/resources/sql/BQAttributeFilterTest/repeatedAttributeFilterUnaryIsNotNull.sql

```diff
@@ -0,0 +1,10 @@
+
+    SELECT
+        name      
+    FROM
+        ${ENT_condition}      
+    WHERE
+        EXISTS (SELECT
+            *          
+        FROM
+            UNNEST(['foo', 'bar', 'baz', vocabulary]))
```

Comment:
I have no idea what this SQL means. Shouldn't there be and `IS NOT NULL` or something? Does it not work correctly when applied to a repeated field?

Reply:
from discussion - this purely a test construct. 
real SQL is of the form

```    
SELECT
        name      
    FROM
        ${ENT_person}      
    WHERE
        EXISTS (SELECT
            *          
        FROM
            UNNEST(consequence))
```    
here consequence is the name of the repeated (not nullable) field. 

repeated fields are not nullable. an empty value is empty array. so the unnest would produce no results (i.e. not exist)